### PR TITLE
Added disclaimer/explanation for some features on README.md & Code analisys with CodeQL

### PR DESCRIPTION
**Pull request by PinkDev1**
In a nutshell:
- Exposed accounts must change a _default setting_ to stop being exposed.
- Accounts that used _Picasa_ are hopeless.

Read commit details, and file changes for insight.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,32 @@ It can currently extract:
...
+- Public photos (T)
+- Phones models (T)
+- Phones firmwares (T)
+- Installed softwares (T)
 - Google Maps reviews
 - Possible physical location
 
+The features marked with a **(T)** requiere the target account to have the default setting of `"allow the people you share content with to download your photos and videos. People may still be able to download photos and videos that you've shared with google photos"`. 
+
+Even for a tech-savy person this setting is hard to find, bc of how buried it is, and how much google tried to hide the fact that some albums can be collected regardless of what the account dashboard says:
```
Comment (outime) [inline README.md:23]: I'd personally avoid writing "bc" and other short words. Also I suggest to use capital G for Google on this file.

Comment (mxrch) [reply README.md:23]: I agree with @outime, and avoiding also terms like "big-ass" :smile:.
Can we also move that Fixing section just before the "Thanks" section ? I think the priority of this project is the offensive part and the fixing part is secondary.

Comment (PinkDev1) [reply README.md:23]: Fair enough :p
I'll edit it as soon as possible

**Comment by mxrch** [comment]
Ah and if you uploaded files on catbox, I think you can delete the pictures in res/ and the res/  folder since you doesn't use them

**Comment by PinkDev1** [comment]
9f5085b 
> The images have also been uploaded to /res/, just in case something happens to the files in files.catbox.moe
I.e. if catbox were to shut down, or ban the project for any arbitrary reason

**Comment by PinkDev1** [comment]
Done! how about now?

**Comment by mxrch** [comment]
How about now ? I think we should also remove the local pictures, res and resources is very close names and the web archives bots has already archived the project, so we don't have to keep these images locally, don't worry :smile:

**Comment by PinkDev1** [comment]
Picasa shut down in 2016. The settings I mention are specific to a Google Photos account. Only the very last comment (the one on issue #10) is related to Picasa, hence why I named the pictures that way.

I might be pretty biased against Google, but it would not be correct to hide the fact that Google **lies** to their users when they visit the Account Dashboard page. Please consider undo-ing that commit

**Comment by mxrch** [comment]
I'm just confused about this picture : https://de.catbox.moe/ufqc9g.jpg
Isn't it on Google Photos ? So I don't understand the link between this screenshot and Picasa, apart the fact that some pics in the Albums Archive also comes from Google Photos or at least can be managed from there. In my case I have only one picture on it managable on Google Photos.
Also, I don't think people's accounts who linked there Picasa account to something are hopeless, because we can delete our pictures on the Album Archives.. so I'm pretty lost about it.

**Comment by PinkDev1** [comment]
> I don't understand the link between this screenshot and Picasa

That screenshot was taken from the [Google Account Dashboard](https://myaccount.google.com/dashboard), and it has no relation to Picasa. It just shows the status of Google Photos (first) and the status the AlbumArchive (second). Obviously, these status are totally innacurrate (`0 public`), since GHunt is capable of pulling down the photos from the AlbumArchive anyway.


> I don't think people's accounts who linked there Picasa account to something are hopeless, because we can delete our pictures on the Album Archives

That's true! I forgot about that. I said "hopeless" previously becouse there is no setting to make ex-Picasa photos not-public, but they may be able to delete ex-Picasa photos from their Google Archive. I can't test this however, since my account never used Picasa; but it's probably as easy as deleting any other album :p


Also, you'll notice that on commit 7fb2261 I added a file called `code-analysis.yml`. What this will do is create a Github Workflow, so that every time a commit is pushed to the repository, CodeQL will analyse the code in search of security vulnerabilities (We wouldn't want a vulnerability in a hacking tool!).
The results of the CodeQL analysis will be visible from the _Actions_ tab.

**Comment by mxrch** [comment]
@PinkDev1 okay I uncommit this but please take a more neutral position for the readme, GHunt is not a tool against Google, it just abuse of the misconfig, you see, and don't use terms like "bullshit" etc in the filenames or the text too, even if you hate Google 😄 
How do you uncommit something in a PR ?
Do I must create a checkout branch with your fork in CLI like when you want to add a commit, and then do the rebase ?
Just to know how you do that and if there is more simple ways (not a problem if not)

**Comment by mxrch** [comment]
And the CodeQL is a good idea yes !

**Comment by PinkDev1** [comment]
> How do you uncommit something in a PR 

This one took me a while, since I mostly use the GUI of _Github Desktop_..
The commit you made was pushed directly onto the _master_ branch of my fork, so to revert it I just had to do a rebase to f47ce5b.
I used `git push --force` because you probably don't have a local copy of my fork, so it shouldn't have caused any problems.
I'll add back the CodeQL and the Google Maps comment shortly

**Comment by PinkDev1** [comment]
All done and ready to merge :)

**Comment by mxrch** [comment]
Perfect !

**Pull request merged**